bittensor/utils/async_substrate_interface.py

```diff
@@ -770,11 +772,13 @@ def __init__(
         """
         self.chain_endpoint = chain_endpoint
         self.__chain = chain_name
-        options = {
-            "max_size": 2**32,
-            "write_limit": 2**16,
-        }
-        self.ws = Websocket(chain_endpoint, options=options)
+        self.ws = Websocket(
+            chain_endpoint,
+            options={
+                "max_size": 2**32,
+                "write_limit": 2**16,
+            },
+        )
```

Comment:
maybe consider using `try... expect...` wrapper around `Websocket(...)` to be able to catch the error to be good